### ISSUE TITLE
Variational Auto-Encoder loss function

### ISSUE THREAD

**Issue by batizhou** [aymericdamien/TensorFlow-Examples#316]
In the example of Variational Auto-Encoder

# KL Divergence loss
kl_div_loss = 1 + z_std - tf.square(z_mean) - tf.exp(z_std)
kl_div_loss = -0.5 * tf.reduce_sum(kl_div_loss, 1)

I don't understand how this formula is come. Can someone help explain it?
